lib/store/my_user.dart

```diff
@@ -139,8 +139,8 @@ class MyUserRepository implements AbstractMyUserRepository {
     }
 
     if (!_blocklistLocal.isEmpty) {
-      final List<RxUser?> users =
-          await Future.wait(_blocklistLocal.blocked.map(_userRepo.get));
+      final List<RxUser?> users = await Future.wait(
+          _blocklistLocal.blocked.map((e) async => _userRepo.get(e)));
```

Comment:
Т.к. [форматтер достаточно глупый](https://dart.dev/effective-dart/style#consider-changing-your-code-to-make-it-more-formatter-friendly), иногда добавления запятых могут помочь ему достичь чуть более приятный результат. Например, здесь он добавил целых 4 пробела. А если добавить запятую, то будет всего 2 пробела, станет чуть проще для понимания код:

```suggestion
      final List<RxUser?> users = await Future.wait(
          _blocklistLocal.blocked.map((e) async => _userRepo.get(e)),);
```